File: lv2/12985.py
'''
예상 대진표
https://school.programmers.co.kr/learn/courses/30/lessons/12985
'''
from math import ceil
def solution(n,a,b):
  answer = 0
  while a != b:   # 대진이 반복될 때 값을 2로 나눈 값들의 올림 값들이 같아지면 둘은 동일한 곳에서 대진을 이루게 됨
    answer += 1
    a = ceil(a/2)
    b = ceil(b/2)
  return answer

File: lv2_js/12985.js
`
예상 대진표
https://school.programmers.co.kr/learn/courses/30/lessons/12985
`
function solution(n, a, b) {
  let roundCnt = 0;
  for (let i = 1; i <= Math.log2(n); i++) { //log 2 의 n 번만큼 대진이 진행되고
    a = Math.ceil(a / 2);
    b = Math.ceil(b / 2);
    if (a === b) {  // 2로 나눈 값을 올림해서 값이 같으면 둘이 같은 대진임
      roundCnt = i;
      break;
    }
  }
  return roundCnt;
}